src/test/scala/me/rexim/github/EtagPollingSpec.scala

```diff
@@ -0,0 +1,34 @@
+package me.rexim.issuestant.github
+
+import org.scalatest._
+import me.rexim.issuestant.mock._
+
+import org.http4s._
+import org.http4s.client._
+import org.http4s.util._
+
+class EtagPollingSpec extends FlatSpec with Matchers {
+  behavior of "EtagPolling"
+
+  it should "reuse etag header for the service response" in {
+    val etagName = CaseInsensitiveString("Etag")
```

Comment:
Yep http://http4s.org/v0.15/api/org/http4s/headers/index.html Use the shit outta them